integration/database_sql_mode_test.go

```diff
@@ -0,0 +1,150 @@
+package integration
```

Comment:
Thank you for adding an integration test! We really appreciate this.

Any feedback around this, please let us know.